server/src/main/java/org/eclipse/openvsx/entities/ExtensionVersion.java

```diff
@@ -70,13 +62,16 @@
     @Column(length = 2048)
     String description;
 
-    @ElementCollection
+    @Column
```

Comment:
column length = 255 is the default, so I omitted it

Reply:
I would set a higher limit for such string lists, like you did for dependencies and bundled extensions.

Reply:
I set now _all_ fields to 2048.